jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/configuration/HandleResourceRunSteps.java

```diff
@@ -178,7 +178,7 @@ public void waitForSaveButton() {
     @And("^I enter attribute in metaData with key as \"(.*)\" and value as \"(.*)\"$")
     public void enterAttribute(String key, String value) {
         jwalaUi.clickWhenReady(By.xpath("(//pre[contains(@class, 'CodeMirror-line')]//span[text()='{'])[1]"));
-        jwalaUi.doubleClick();
+        jwalaUi.backSpace();
```

Comment:
I still don't understand how replacing a double click with a backspace fixes something - this seems like a totally different test now

Reply:
Actually, we do delete "{" tag by selecting it with double click and send-keys over it. With new capabilities provided it was not functioning well. So, i removed the double click click functionality with backspace function + send keys which is exactly required for the operation.

Reply:
Select the "{" text and copy the string(doubleclick+sendkeys) is replaced with delete the text "{" and copy string (backspace+sendkeys) function